### PR TITLE
Index Linux repositories: add retry

### PR DESCRIPTION
**Pull request by webrouse**
Slack: https://keboola.slack.com/archives/C02JAQXLY92/p1645199647555099

Changes:
- RPM repository: temp files are deleted.
- All Linux repositories: added retry

### CODE REVIEW

File: build/package/linux/rpm/index.sh [hunk 1]
```diff
@@ -10,8 +10,13 @@ set -o pipefail         # Use last non-zero exit code in a pipeline
 echo -e "$RPM_KEY_PUBLIC"  | gpg --import --batch
 echo -e "$RPM_KEY_PRIVATE" | gpg --import --batch
 
-# Index
 cd /packages/rpm
+
+# Clear
+rm -rf .repodata || true
+rm -rf repodata.old.* || true
```
Comment (webrouse) [inline build/package/linux/rpm/index.sh:17]: Zostava tam nejaky neporiadok:
image

File: build/package/linux/index.sh [hunk 2]
```diff
@@ -6,6 +6,48 @@ set -o nounset          # Disallow expansion of unset variables
 set -o pipefail         # Use last non-zero exit code in a pipeline
 #set -o xtrace          # Trace the execution of the script (debug)
 
+# https://gist.github.com/sj26/88e1c6584397bb7c13bd11108a579746
+function retry {
+  local retries=$1
+  shift
+
```
Comment (webrouse) [inline build/package/linux/index.sh:13]: Retry na indexovanie linux repozitarov, uvidime ci to bude stacit.

**Comment by webrouse** [comment]
(este fixnem jeden preklep )

File: build/package/linux/rpm/index.sh [hunk 3]
```diff
@@ -10,9 +10,14 @@ set -o pipefail         # Use last non-zero exit code in a pipeline
...
+
+# Index
 createrepo --update  .
 
 # Sign
 gpg --yes --detach-sign --armor repodata/repomd.xml
+
+# Clear
+rm -rf .repodata || true
+rm -rf repodata.old.* || true
```
Comment (webrouse) [inline build/package/linux/rpm/index.sh:23]: Zostava tam nejaky neporiadok:
image

**Review by JakubMatejka — APPROVED** [review]
👍🤞

**Pull request merged**